src/ReactiveUI.Winforms/ActivationForViewFetcher.cs

```diff
@@ -13,23 +13,25 @@ namespace ReactiveUI.Winforms
 {
     public class ActivationForViewFetcher : IActivationForViewFetcher, IEnableLogger
     {
+        private bool? _isDesignModeCache = null;
+        
         public int GetAffinityForView(Type view)
         {
             return (typeof(Control).GetTypeInfo().IsAssignableFrom(view.GetTypeInfo())) ? 10 : 0;
         }
 
         public IObservable<bool> GetActivationForView(IActivatable view)
         {
-            // Check for design time
-            if (LicenseManager.UsageMode == LicenseUsageMode.Designtime) {
-                return Observable<bool>.Default;
-            }
-
             // Startup: Control.HandleCreated > Control.BindingContextChanged > Form.Load > Control.VisibleChanged > Form.Activated > Form.Shown
             // Shutdown: Form.Closing > Form.FormClosing > Form.Closed > Form.FormClosed > Form.Deactivate
             // https://docs.microsoft.com/en-us/dotnet/framework/winforms/order-of-events-in-windows-forms
 
             if (view is Control control) {
+
+                if (GetCachedIsDesignMode(control)) {
```

Comment:
I'm not sure why I added the Designtime check there. 
I mean in Winforms Designer the events are executed and the control was considered active and that led to errors, because e.g. the ViewModel is null at design time.